roles/skydive/tasks/provision.yml

```diff
@@ -0,0 +1,72 @@
+---
+- name: Login As Super User
+  shell: "oc login -u {{ admin_user }} -p {{ admin_password }}"
+  when: cluster=="openshift"
+    and admin_user is defined
+    and admin_password is defined
+
+- name: Check if namespace "{{ skydive_namespace }}" exists
+  shell: "kubectl get ns | grep -w {{ skydive_namespace }} | awk \'{ print $1 }\'"
+  register: ns
+
+- name: Create {{ skydive_namespace }} namespace on the kubernetes cluster
+  shell: kubectl create namespace {{ skydive_namespace }}
+  when: ns.stdout != skydive_namespace
+    and cluster=="kubernetes"
+
+- name: Create {{ skydive_namespace }} project on the openshift cluster
+  shell: oc new-project {{ skydive_namespace }}
+  when: ns.stdout != skydive_namespace
+    and cluster=="openshift"
+
+- name: Render skydive deployment yaml for kubernetes
+  template:
+    src: kubernetes-skydive.yaml
+    dest: /tmp/skydive.yaml
+  when: cluster=="kubernetes"
+
+- name: Render skydive deployment yaml for openshift
+  template:
+    src: openshift-skydive.yaml
+    dest: /tmp/skydive.yaml
+  when: cluster=="openshift"
+
+- name: Deploy skydive for openshift
```

Comment:
this is redundant with the below task.  You can do an apply with both clients.

Reply:
I think you can't because the openshift yaml is:

```yaml
apiVersion: v1
kind: Template
metadata:
  name: skydive
...
```
If you try to run `kubectl apply -f` on this file it will fail.

Reply:
What's the purpose of creating the template?  Why not use the same yaml?

Reply:
I used the supported deployment for every environment by the skydive team

openshift (https://github.com/skydive-project/skydive/tree/master/contrib/openshift)
kubernetes (https://github.com/skydive-project/skydive/tree/master/contrib/kubernetes)

We can use the kubernetes one for both, I just wanted to use the official way for every one of the platforms.

Reply:
Using a template without variables and yaml resources separated with `---` are identical.  Looks like there's a difference between the two with resources.  Let's keep as is.